jetbrains-rider/src/software/aws/toolkits/jetbrains/services/lambda/dotnet/DotNetSamProjectGenerator.kt

```diff
@@ -167,7 +167,7 @@ class DotNetSamProjectGenerator(
         val progressManager = ProgressManager.getInstance()
         progressManager.runProcessWithProgressSynchronously(
             {
-                samSettings.template.build(context.project, projectNameField.text, samSettings.runtime, samSettings.schemaParameters, outDirVf)
+//                samSettings.template.build(context.project, projectNameField.text, samSettings.runtime, samSettings.schemaParameters, outDirVf)
```

Comment:
line commented out, should it be there still?

Reply:
whoops, guess I need to test Rider :D

Reply:
Fixed Rider